tools/tools.qmd

```diff
@@ -1,187 +1,93 @@
-    <td>[MathematicaStan](https://github.com/stan-dev/MathematicaStan/blob/master/README.org){target="_blank"}
-    </td>
-  </tr>
-  <tr>
-    <td>MATLAB</td>
-    <td>
-      [MatlabStan](https://github.com/brian-lau/MatlabStan/wiki){target="_blank"}
-    </td>
-  </tr>
-  <tr>
-    <td>Python</td>
-    <td>
-      [CmdStanPy](https://mc-stan.org/cmdstanpy){target="_blank"}*,
-      ([PyStan](https://mc-stan.org/pystan){target="_blank"} - legacy - not maintained)
-    </td>
-  </tr>
-  <tr>
-    <td>R</td>
-    <td>
-      [CmdStanR](https://mc-stan.org/cmdstanr){target="_blank"}*,
-      [RStan](https://mc-stan.org/rstan){target="_blank"}
-    </td>
-  </tr>
-  <tr>
-    <td>shell</td>
-    <td>[CmdStan](https://mc-stan.org/cmdstan){target="_blank"}</td>
-  </tr>
-  <tr>
-    <td></td>
-    <td><b>*Note:</b> CmdStanPy and CmdStanR are recommended for new users
-  </tr>
-</table>
-
-## High-level Interfaces
-<table style="width: 85%;">
-  <tr style="background-color: var(--stan-highlight);">
-    <th style="width: 25%;">Language</th>
-    <th style="width: 75%;">Stan versions of common models</td>
-  </tr>
-  <tr>
-    <td>R</td>
-    <td>
-      [BRMS](https://paul-buerkner.github.io/brms/){target="_blank"}<br>
-      Specify and fit multivariate and multilevel models in Stan using extended R-formula syntax.
-    </td>
-  </tr>
-  <tr>
-    <td>R</td>
-    <td>
-      [RStanArm](https://mc-stan.org/rstanarm){target="_blank"}<br>
-      Pre-compiled Stan versions of existing R model functions.
-    </td>
-  </tr>
-</table>
-
-## Visualization and Validation 
-<table style="width: 85%;">
-  <tr style="background-color: var(--stan-highlight);">
-    <th style="width: 25%;">Language</th>
-    <th style="width: 75%;">Validate, visualize, and compare fitted models</th>
-  </tr>
-  <tr>
-    <td>Julia</td>
-    <td>[ArviZ.jl](https://julia.arviz.org/ArviZ/stable/){target="_blank"}</td>
-  </tr>
-  <tr>
-    <td>Python</td>
-    <td>[ArviZ](https://python.arviz.org/en/stable/){target="_blank"}</td>
-  </tr>
-  <tr>
-    <td>R</td>
-    <td>[Bayesplot](https://mc-stan.org/bayesplot/){target="_blank"} - plot model outputs</td>
-  </tr>
-  <tr>
-    <td></td><td>[loo](https://mc-stan.org/loo/){target="_blank"} - leave-one-out cross validation</td>
-  </tr>
-  <tr>
-    <td></td><td>[posterior](https://mc-stan.org/posterior){target="_blank"} - get model estimates</td>
-  </tr>
-  <tr>
-    <td></td><td>[projpred](https://mc-stan.org/projpred){target="_blank"} - check model predictions</td>
-  </tr>
-  <tr>
-    <td>Web browser</td>
-    <td>[MCMC Monitor](https://github.com/flatironinstitute/mcmc-monitor){target="_blank"}</td>
-  </tr>
-</table>
+# Stan Toolkit
+
+A curated collection of tools and interfaces to help you work effectively with Stan across various programming environments and stages of your modeling workflow.
+
+---
+
+## Language-Specific Stan Interfaces
+
+*Write, compile, and run Stan models directly within your programming environment.*
```

Comment:
Feels maybe a bit repetitive with the install page, but that's not necessarily a bad thing. We should probably link to the install matrix still

Reply:
excellent suggestion.   added cross links between pages.